sql/sql_connect.cc

```diff
@@ -158,63 +158,29 @@ void free_thread_stats(THREAD_STATS* thread_stats)
   my_free((char *) thread_stats);
 }
 
-void init_user_stats(USER_STATS *user_stats,
-                     const char *user,
-                     const char *priv_user,
-                     uint total_connections,
-                     uint total_ssl_connections,
-                     uint concurrent_connections,
-                     time_t connected_time,
-                     double busy_time,
-                     double cpu_time,
-                     ulonglong bytes_received,
-                     ulonglong bytes_sent,
-                     ulonglong binlog_bytes_written,
-                     ha_rows rows_fetched,
-                     ha_rows rows_updated,
-                     ha_rows rows_read,
-                     ulonglong select_commands,
-                     ulonglong update_commands,
-                     ulonglong other_commands,
-                     ulonglong commit_trans,
-                     ulonglong rollback_trans,
-                     ulonglong denied_connections,
-                     ulonglong lost_connections,
-                     ulonglong access_denied_errors,
-                     ulonglong empty_queries)
+static void init_user_stats(USER_STATS *user_stats)
```

Comment:
I suggested to remove only the always-zero args, not all the args. The current way is also fine, but then this whole function should be replaced by a memset.

If you choose to restore the non-zero args, then use memset, and then set only those args.